### ISSUE TITLE
multi account

### ISSUE THREAD

**Issue by Axel3026** [farshadz1997/Microsoft-Rewards-bot#57]
Is there a way to run many accounts at the same time so it farms faster for good computers
The only one that occurred to me was to create .bat with the --accounts

**Comment by zera4real** [comment]
Yeah, I also want something like that.

**Comment by Axel3026** [comment]
@zera4real  for the moment you could use the bats -

echo off
cd E:\Downloads\GITHUB REPOSITORY\MicrosoftRewardsBotAxel-WFD
start MicrosoftRewardsBot.py --session --headless --fast --accounts

**Comment by zera4real** [comment]
@Axel3026 okay, thanks

**Comment by zera4real** [comment]
Also, can we have a discord server?

**Comment by farshadz1997** [comment]
It's pretty difficult to do it on bot because it requires to change the structure of all functions and may create so many bugs, I think use bat config is better. Take a look at #37.

**Comment by LPridemore** [comment]
There was a great bot here before. It was built on the old Sealemar bot. It was fully upgraded though, it could handle multiple accounts. 1,000 accounts = no problem at all.  I watched my friend run 1500 accounts on it. 
AND IT RAN THEM SIMULTANEOUSLY TOO. Not one after another like this bot does.  BUT that is why MS has not taken this bot down most likely. People won't be making $150,000.00 a year selling codes from a bot that can only do one account at a time.  

The dev deleted it. I do know someone who has the last version of it. I'm sure farshadz1997 could work with that one, update it to the new MS format or whatever you guys all the changes MS makes all the time to try to stop bots.

**Comment by zuko726** [comment]
> There was a great bot here before. It was built on the old Sealemar bot. It was fully upgraded though, it could handle multiple accounts. 1,000 accounts = no problem at all. I watched my friend run 1500 accounts on it. AND IT RAN THEM SIMULTANEOUSLY TOO. Not one after another like this bot does. BUT that is why MS has not taken this bot down most likely. People won't be making $150,000.00 a year selling codes from a bot that can only do one account at a time.
> 
> The dev deleted it. I do know someone who has the last version of it. I'm sure farshadz1997 could work with that one, update it to the new MS format or whatever you guys all the changes MS makes all the time to try to stop bots.

are you talking about this one?
https://web.archive.org/web/20180611033206/https://github.com/sealemar/BingRewards

**Comment by LPridemore** [comment]
@zuko726 No. I'm talking about a bot builder that took that sealemar bot and vastly improved it. It was amazing. Could handle all the accounts you want to add to it.
